### PR TITLE
Replace the config file with yaml inputs

### PR DESCRIPTION
**Pull request by harish-24**
Patch replaces specified yaml inputs(mnt dirs) in local.config file

Signed-off-by: Harish <harish@linux.vnet.ibm.com>

### CODE REVIEW

File: fs/xfstests.py [hunk 1]
```diff
@@ -118,13 +118,17 @@ def setUp(self):
...
+            cfg_file = os.path.join(self.teststmpdir, 'local.config')
+            cmd_template = 'sed -i "s/export %s=.*/%s/g" %s'
+            line = (l_template % ('TEST_DEV', self.devices[0])).replace('/', '\/')
+            process.system(cmd_template % ('TEST_DEV', line, cfg_file), shell=True)
+            line = (l_template % ('SCRATCH_DEV', self.devices[1])).replace('/', '\/')
+            process.system(cmd_template % ('SCRATCH_DEV', line, cfg_file), shell=True)
+            line = (l_template % ('TEST_DIR', self.test_mnt)).replace('/', '\/')
+            process.system(cmd_template % ('TEST_DIR', line, cfg_file), shell=True)
+            line = (l_template % ('SCRATCH_MNT', self.scratch_mnt)).replace('/', '\/')
+            process.system(cmd_template % ('SCRATCH_MNT', line, cfg_file), shell=True)
```
Comment (narasimhan-v) [inline fs/xfstests.py]: Wouldn't a file write from python be better ?

Comment (harish-24) [reply fs/xfstests.py]: sure, let me change accordingly

File: fs/xfstests.py [hunk 2]
```diff
@@ -118,13 +118,21 @@ def setUp(self):
...
-            process.system('sed -i "s/export SCRATCH_DEV=.*/%s/g" %s' %
-                           (line, os.path.join(self.teststmpdir, 'local.config')), shell=True)
+            cfg_file = os.path.join(self.teststmpdir, 'local.config')
+            with open(cfg_file, "r") as sources:
+                lines = sources.readlines()
+            with open(cfg_file, "w") as sources:
+                for line in lines:
+                    if line.startswith('export TEST_DEV'):
+                        sources.write(re.sub(r'export TEST_DEV=.*', 'export TEST_DEV=%s'%self.devices[0], line))
+                    if line.startswith('export TEST_DIR'):
```
Comment (narasimhan-v) [inline fs/xfstests.py]: `elif` is better

Comment (harish-24) [reply fs/xfstests.py]: done

**Pull request merged**